### PR TITLE
RDM-11548: Ignore the CREATOR role from the existing case user relationship

### PR DESCRIPTION
**Pull request by rajaram-nagarajan**
### JIRA link ###

https://tools.hmcts.net/jira/browse/RDM-11548

### Change description ###

Ignored the creator role from the existing case user relationship

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by MSancaktutar** [comment]
@rajaram-nagarajan  - This PR should be based on master. We'll release this with CI/CD.

**Review by MSancaktutar — APPROVED** [review]
Change should be based on `master` branch, otherwise looks good.

**Comment by rajaram-nagarajan** [comment]
> @rajaram-nagarajan - This PR should be based on master. We'll release this with CI/CD.

Sure, Mutlu. I just rebased onto master.

**Review by skandala-cgi — APPROVED** [review]
Looks Good.

File: src/aat/resources/features/F-105 - Add Case-Assigned Users and Roles/With Organisation Context/S-105.18/S-105.18.td.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+  "specs": [
+    "Richard - who can create a case",
+    "Dil - who is to add some case role assignment for a case",
+    "Olawale - with an active solicitor profile and valid User ID",
+    "is made from an authorised application, by Dil, with the Case ID of C1, User ID of Olawale, proper Case Role CR-2 and the Organisation ID of Olawale"
+  ],
+
+  "users": {
+    "_extends_": "F-105_Users"
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-105 - Add Case-Assigned Users and Roles/With Organisation Context/S-105.18/S-105.18.td.json:15]: put here the user Dil, to make it consistent with the injected token in the request header

Comment (rajaram-nagarajan) [reply src/aat/resources/features/F-105 - Add Case-Assigned Users and Roles/With Organisation Context/S-105.18/S-105.18.td.json:15]: Corrected

**Comment by MSancaktutar** [comment]
Merged as confirmed.

**Pull request merged**